python/adelphi/integration-tests/adelphi-docker-tests.sh

```diff
@@ -59,17 +74,26 @@ pip install ../../adelphi
 mkdir -p output logs
 
 # clear previous results
-rm output/* logs/*
+rm -rf output/* logs/*
 
 # C* versions to test with (these must be available Docker images in dockerhub)
 versions=(2.1.22 2.2.19 3.0.23 3.11.9 4.0-beta3)
+
+allPass=""
 for v in "${versions[@]}"
 do
 	start $v
 	create_schema $v
-	anonymize $v
+	export_schema $v
 	compare $v
```

Comment:
suggestion: call `compare` twice (once for each schema output type - stdout and file) then you can simplify your compare function implementation by checking a single file and a single return code at a time

Reply:
Hmmmm, interesting idea.  I'm not sure how much that actually buys me though.  I still have to put that cumbersome logic to gather two return values (one from each schema comparison) and check to make sure they're both zero before showing the "All comparisons matched..." message for a given C* version.  Doesn't the approach you're suggesting just move that logic to this do block rather than doing it in the compare function?

I kinda like where you're headed with this, but if at the core we're just moving around where that comparison of two return values happens I don't know that it helps much.